parm/config/config.landanl

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -x
+
+########## config.landanl ##########
+# configuration common to all land analysis tasks
+
+echo "BEGIN: config.landanl"
+
+obs_list_name=gdas_land_adpsfc_only.yaml
+if [[ "${cyc}" == "18" ]]; then
+    obs_list_name=gdas_land_prototype.yaml
+fi
+
+export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/land/obs/config/
+export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/land/obs/lists/${obs_list_name}
+export LANDVARYAML=${HOMEgfs}/sorc/gdas.cd/parm/land/letkfoi/letkfoi.yaml
+export FV3JEDI_FIX=${HOMEgfs}/fix/gdas
```

Comment:
Is this going to be used?  @RussTreadon-NOAA had removed its reference.

Reply:
we should be consistent, I don't think it's needed now as long as the fix directory is linked

Reply:
@aerorahul I checked that @RussTreadon-NOAA still kept this reference in the `config.atmanl`, and similar for the `config.aeroanl`. Therefore, we keep this reference here for consistency.